chart/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    {{- if (.Values.teamName) }}
+    team: {{ .Values.teamName }}
+    {{- end}}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        {{- if (.Values.teamName) }}
+        team: {{ .Values.teamName }}
+        {{- end}}
+        version: {{ .Chart.Version | replace "+" "_" }}-{{ .Release.Revision }}
+      annotations:
+        prometheus.io/scrape: {{ .Values.prometheusScrape | quote }}
+        prometheus.io/port: {{ .Values.prometheusPort | quote }}
```

Comment:
Same as before, in our case we scrape using Prometheus, but most of the charts leave it configurable: https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L50-L52

Reply:
`values.yaml`:
```
annotations:
  prometheus.io/scrape: "true"
  prometheus.io/port: "9101"
```

deployment templates:
```
  annotations:
    {{- if (.Values.annotations) }}
{{ toYaml .Values.annotations| indent 4 }}
    {{- end}}
```